apps/dgst.c

```diff
@@ -413,6 +413,28 @@ int dgst_main(int argc, char **argv)
     return ret;
 }
 
+const char * newline_escape_filename(const char *file, int * backslash)
```

Comment:
Please be consistent with * and spaces.  Generally, the space after a pointer star is omitted.
There are two here with a following and one without.

Why isn't the function static?

Reply:
Will make the pointer and spacing consistent. 

My initial thought on why this function is not static was there might be an opportunity for use in other places.  If this is not realistic then I can make the function static.  Thoughts?

Reply:
If it should become a function for general use, then it should end up in a separate file, or in apps.c

Reply:
Thank you. For this next commit I will keep the function static. I will be on the lookout for other use cases for this function to move it out to apps.c.